spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -74,4 +74,66 @@
     include_examples 'Unauthenticated User'
   end
 
+  describe 'GET #info' do
+    subject(:http_request) { get :info, params: params}
+    
+
+    context 'When user is authenticated' do
+      include_context 'Authenticated User'
+
+      context 'When fetching an existing book' do
+        before do
+          open_library_stubbed_service = instance_double(OpenLibraryService)
+          allow(open_library_stubbed_service).to receive(:book_info).and_return(
+            JSON.parse(File.read('spec/support/fixtures/open_library_service_parsed_response.json')).symbolize_keys)
+          allow(OpenLibraryService).to receive(:new).and_return(open_library_stubbed_service)
+          http_request
+        end
+
+        let(:params) { { isbn: '0385472579' } }
```

Comment:
Can you faker here? Do not forgot you are mocking the response from service you can do what you want. 
However you are maybe right to want to keep the flow... I am not decided on this point.